public/dist/index.html

```diff
@@ -1,19 +1,29 @@
 <!DOCTYPE html>
 <html>
+
+<head>
+  <link rel="stylesheet" type="text/css" href="/style.css"/>
+</head>
     
 <body>
-  <div style="float: right; padding-right: 5%; padding-left: 30px;">
-    <div id="product-module" style="padding-left: 25px;"></div>
-    <div id="inventory-module" style="padding-bottom: 50px;"></div>
-    <div id="reviews-module" style="padding-left: 25px;"></div>
+
+  <div class="menu">
+    <div class="menu-right">
+      <div id="product-module"></div>
+      <div id="inventory-module"></div>
+      <div id="reviews-module"></div>
+    </div>
+  
+    <div class="gallery">
+      <div id="photo-module"></div>
+    </div>
   </div>
-  <div id="photo-module" style="width: 90%; padding-left: 30px; "></div>
   
 
-  <script src="http://localhost:3008/product-bundle.js"></script>
-  <script src="http://localhost:3000/bundle-photo-module.js"></script>
-  <script src="http://localhost:3004/bundle.js"></script>
-  <script src="http://localhost:3003/bundle.js"></script>
+  <script src="http://3.128.76.194:3008/product-bundle.js"></script>
```

Comment:
Just a note: I was wrong about having to use the deployed service's ip address + bundle.js. You can use s3's bundle.js so long as it's up to date with your service repo. S3 will reduce load, but it's optional!